cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,49 @@
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+import { VerifyValidator, RemarkMetadataAction } from '@joystream/metadata-protobuf'
+import { metadataToString } from '../../helpers/serialization'
+
+export default class VerifyValidatorAccountCommand extends WorkingGroupsCommandBase {
+  static description = 'Membership lead/worker verifies validator membership profile'
+  static args = [{
+    name: "memberId",
+    required: true,
+    description: 'Membership ID of the validator to verify',
+  },
+  {
+
+    name: "isVerified",
+    required: true,
+    description: 'Verification state of the validator',
+  }
+  ]
+
+  static flags = {
+    ...WorkingGroupsCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    const { memberId, isVerified } = this.parse(VerifyValidatorAccountCommand).args
+
+    const meta = new RemarkMetadataAction({
+      verifyValidator: new VerifyValidator({
+        memberId,
+        isVerified,
+      }),
+    })
+    const message = metadataToString(RemarkMetadataAction, meta)
+
+    const nowGroup = this.group
+
+    await this.setPreservedState({ defaultWorkingGroup: WorkingGroups.Membership })
+    const worker = await this.getRequiredWorkerContext()
+
+    if (!worker) {
+      this.error('Only membership WG lead/worker can perform this command')
+    } else {
+      this.getOriginalApi().tx.membershipWorkingGroup.workerRemark(Number(worker), message)
```

Comment:
`worker` is a type of [GroupMember](https://github.com/chrlschwb/joystream/blob/bcc88ee4ac7f3a1be87e79bd7fb5568e102aa774/cli/src/Types.ts#L90), while you are converting it to Number. Which is obviously incorrect.